src/Microsoft.ML.Core/Prediction/ITrainer.cs

```diff
@@ -56,15 +55,9 @@ public interface ITrainerEx : ITrainer
         /// Whether this trainer could benefit from a cached view of the data.
         /// </summary>
         bool WantCaching { get; }
-    }
-
-    public interface ITrainerHost
-    {
-        Random Rand { get; }
-        int Verbosity { get; }
 
-        TextWriter StdOut { get; }
-        TextWriter StdErr { get; }
+        bool SupportsValidation { get; }
```

Comment:
Do we want XML comments on these properties, like the other properties?

Reply:
Yes.

---
In reply to: [202181399](https://github.com/dotnet/machinelearning/pull/522#discussion_r202181399) [](ancestors = 202181399)